cli/commands/common.go

```diff
@@ -52,6 +52,10 @@ func addStateboardRunningFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVar(&ctx.Running.StateboardOnly, "stateboard-only", false, stateboardOnlyUsage)
 }
 
+func addDisablePrefixFlag(cmd *cobra.Command) {
+	cmd.Flags().BoolVar(&ctx.Running.DisableLogPrefix, "disable-prefix", false, disablePrefixUsage)
```

Comment:
Maybe "--no-log-prefix" (as it was done for docker-compose - https://github.com/docker/compose/pull/7435)